### DISCUSSION TITLE
Support for MSSQL

### DISCUSSION THREAD

**Discussion by willianhf** [prisma/prisma#2427] [Q&A] [answered]
Will Prisma have support to Microsoft SQL Server?

**Comment by ryands17** [comment, marked as answer]
Hey @willianhf :wave: 
Could you open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) regarding this support so that we can look into it.

**Comment by mad-gav** [comment]
Microsoft SQL support would be a game changer. Loads of companies use SQl server and for us to try out prisma we'd need sql server support.

How do i see the list of feature requests?

**Reply by pantharshit00** [reply]
@mad-gav You can see a list of feature request by filtering with `kind/feature` label.

Here is the list: https://github.com/prisma/prisma/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Ffeature+sort%3Areactions-%2B1-desc

**Reply by willianhf** [reply]
You can follow the progress on [this issue](https://github.com/prisma/prisma/issues/2430).

**Comment by ludralph** [comment]
Hi @willianhf,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
